core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2022/model/Timetable.kt

```diff
@@ -62,7 +62,9 @@ public data class Timetable(
         }
         if (filters.searchWord.isNotBlank()) {
             timetableItems = timetableItems.filter { timetableItem ->
-                timetableItem.title.currentLangTitle.contains(filters.searchWord)
+                timetableItem.title.currentLangTitle.lowercase().contains(
```

Comment:
How about using contains(text, ignoreCase = false)?

Reply:
I don't know this option.
But, false is case sensitive. so I use true.